src/components/Staking/StakingConsiderations/index.tsx

```diff
@@ -75,9 +75,18 @@ const StakingConsiderations = ({ page }: StakingConsiderationsProps) => {
     pageData,
     activeIndex,
   } = useStakingConsiderations({ page })
+  
+  const activeStyles = {
+    bg: "var(--eth-colors-offBackground)",
+    color: "var(--eth-colors-chakra-body-text)",
```

Comment:
Similar to above, we should use the `body.base` semantic token here:

```suggestion
    color: "body.base",
```

cc: @nloureiro